FirebaseTestProject/Views/ImagePickerView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ImagePickerView.swift
+//  ImagePickerView
+//
+//  Created by Tim Miller on 8/30/21.
+//
+
+import UIKit
+import SwiftUI
+
+struct ImagePickerView: UIViewControllerRepresentable {
+    typealias SourceType = UIImagePickerController.SourceType
+    
+    var sourceType: SourceType
+    @Binding var selection: UIImage?
+    @Environment(\.dismiss) private var dismiss
+    
+    func makeCoordinator() -> Coordinator {
+        .init(view: self)
+    }
+    
+    func makeUIViewController(context: Context) -> UIImagePickerController {
+        let imagePicker = UIImagePickerController()
+        imagePicker.sourceType = sourceType
+        imagePicker.delegate = context.coordinator
+        imagePicker.allowsEditing = true
+        return imagePicker
+    }
+    
+    func updateUIViewController(_ uiViewController: UIImagePickerController, context: Context) {
+        
```

Comment:
Do we need this for `UIViewControllerRepresentable` conformance?

Reply:
Yes